api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -46,6 +46,12 @@ message RedisProxy {
     // * '{user1000}.following' and '{user1000}.followers' **will** be sent to the same upstream
     // * '{user1000}.following' and '{user1001}.following' **might** be sent to the same upstream
     bool enable_hashtagging = 2;
+
+    // Accept moved and ask redirection errors from upstream redis servers, and retry commands
```

Comment:
nit: is it possible to link to relevant redis docs here?

Reply:
yeah, I'll do that ...